dashboard/src/utils/filterDataset.js

```diff
@@ -0,0 +1,12 @@
+import {dateFromUTCString } from "./constructDate";
+export const filterData = (dataArray, startDate, endDate, controllerValue) => {
+  let res = dataArray.filter((data) => {
+    let datasetDate = dateFromUTCString(`${data.metadata["dataset.created"]}`);
```

Comment:
This doesn't seem like it's going to work, now that [Dave has updated his server](https://github.com/distributed-system-analysis/pbench/pull/2733#discussion_r867519062) to return the `created` value with the ISO timezone offset included.

The `Date()` c'tor won't accept `'2022-02-16T13:21:29+00:00Z'` (that is, with _both_ the `+00:00` and the `Z`).

But, this should work:
```
    let datasetDate = Date(`${data.metadata["dataset.created"]}`);
```